smartsim/_core/utils/timings.py

```diff
@@ -144,21 +144,21 @@ def max_length(self) -> int:
             return 0
         return max(len(value) for value in self._timings.values())
 
-    def print_timings(self, to_file: bool = False) -> None:
+    def print_timings(self, to_file: bool = False, to_stdout: bool = True) -> None:
```

Comment:
nitpick alert! now that you've moved this method onto a timing object, it might be unnecessary to call this `print_timings` instead of just `print`

Reply:
Done